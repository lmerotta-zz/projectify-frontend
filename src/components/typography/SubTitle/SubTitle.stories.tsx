// Button.stories.tsx
import { ComponentProps } from "react";
import SubTitle from "./SubTitle";
import { Story } from "@storybook/react/types-6-0";

// eslint-disable-next-line import/no-anonymous-default-export
export default {
  title: "Typography/SubTitle",
  component: SubTitle,
};

const Template: Story<ComponentProps<typeof SubTitle>> = () => (
  <SubTitle>Sub-title</SubTitle>
);

export const Default = Template.bind({});
