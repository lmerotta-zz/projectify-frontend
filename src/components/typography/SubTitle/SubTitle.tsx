import tw from "twin.macro";

export default tw.h2`text-xl md:text-2xl text-gray-400 font-medium`;
