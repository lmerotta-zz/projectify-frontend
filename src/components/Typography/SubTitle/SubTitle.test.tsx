import { render } from "@testing-library/react";
import SubTitle from "./SubTitle";

describe("SubTitle unit tests", () => {
  it("Renders", () => {
    const { asFragment } = render(<SubTitle>My SubTitle</SubTitle>);

    expect(asFragment()).toMatchSnapshot();
  });
});
