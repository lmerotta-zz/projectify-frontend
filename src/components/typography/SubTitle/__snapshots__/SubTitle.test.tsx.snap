// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitle unit tests Renders 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-size: 1.25rem;
  line-height: 1.75rem;
  --tw-text-opacity: 1;
  color: rgba(156, 163, 175, var(--tw-text-opacity));
  font-weight: 500;
}

@media (min-width: 768px) {
  .emotion-0 {
    font-size: 1.5rem;
    line-height: 2rem;
  }
}

<h2
    class="emotion-0"
  >
    My SubTitle
  </h2>
</DocumentFragment>
`;
